coalib/parsing/Globbing.py

```diff
@@ -252,31 +252,37 @@ def _absolute_flat_glob(pattern):
     return
 
 
-def _iter_relative_dirs(dirname):
+def _iter_relative_dirs(dirname, ignored_globs=None):
     """
     Recursively iterates subdirectories of all levels from dirname
 
+    :param ignored_globs: List of globs to ignore when matching files
```

Comment:
maintain the order of param definition according to the argument..
```Cython
:param dirname:          Directory name
:param ignored_globs: List of globs to ignore when matching files
:return:                         Iterator that yields files and directory from the given dir
                                    and all it's (recursive) subdirectories
```
Also maintain the alignment..